### PR TITLE
Plugin dependencies support

### LINKED ISSUE #36: Plugin dependencies support
Mqtt, Stomp plugins are dependencies of `rabbitmq_test` now (in `master` branch) and will be started during unit test run.
They are started with the same default port for both `rabbit` and `hare` nodes. Which cause node startup failure.

There are several test failing due to plugin dependencies.

### PR DESCRIPTION
**Pull request by hairyhum**
Fixes #36

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -150,12 +150,15 @@ clean-qpid-testsuite:
...
 prepare: test-dist create_ssl_certs
+	$(verbose) rm -rf plugins/rabbitmq_metronome*
+	$(verbose) rm -rf plugins/rabbitmq_amqp1_0*
 	$(verbose) $(MAKE) \
 		RABBITMQ_NODENAME=hare \
 		RABBITMQ_NODE_IP_ADDRESS=0.0.0.0 \
 		RABBITMQ_NODE_PORT=${TEST_HARE_PORT} \
 		RABBITMQ_SERVER_START_ARGS=$(HARE_BROKER_OPTIONS) \
 		RABBITMQ_CONFIG_FILE=/does-not-exist \
+		RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS="-rabbitmq_stomp tcp_listeners [61614] -rabbitmq_mqtt tcp_listeners [1884] -rabbitmq_web_mqtt tcp_config [{port,15676}] -rabbitmq_web_stomp port 15685 -rabbitmq_management listener [{port,15673}]" \
```
Comment (binarin) [inline Makefile:161]: Same ports for rabbit and hare?

File: Makefile [hunk 2]
```diff
@@ -150,12 +150,15 @@ clean-qpid-testsuite:
 	$(gen_verbose) rm -rf qpid_testsuite
 
 prepare: test-dist create_ssl_certs
+	$(verbose) rm -rf plugins/rabbitmq_metronome*
```
Comment (binarin) [inline Makefile:153]: I like this solution. Because initially I decided to try to fix `metronome` instead )

File: Makefile (hunk 1 above)
Comment (hairyhum) [inline Makefile:161]: Which ones? Did I missed something?

Comment (binarin) [inline Makefile:161]: Sorry, my mistake - I misinterpreted what this patch is doing.

**Pull request merged**